README.md

```diff
@@ -262,6 +262,13 @@ credentials once from your workstation. After this is done, Travis CI will be ab
 repeat the `make deploy` step from a privileged account any time you change the IAM policies templates in
 `iam/policy-templates/`.
 
+#### Authorizing the event relay
+
+Environment variables provide the AWS credentials needed to relay events originating from supported cloud platforms
+outside of AWS. Run `scripts/create_aws_event_relay_user.py` to create an AWS IAM user with the appropriate
+restricted access policy. The access key id and secret access key are obtained from the AWS console, and need to be
+populated into AWS SSM parameters by running the script `scripts/set_event_relay_parameters.py`.
```

Comment:
In `scripts/set_event_relay_parameters.py`, passing secrets as command line parameters is insecure. Instead, please use the IAM method for getting the key ID and secret key.

Reply:
Fixed